src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/LoongArch/LoongArchBase.cs

```diff
@@ -0,0 +1,198 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.LoongArch
+{
+    /// <summary>
+    /// This class provides access to the LoongArch base hardware instructions via intrinsics
+    /// </summary>
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class LoongArchBase
```

Comment:
* Rename this file as `LoongArchBase.cs`, is it OK? 
Or Just name this file as `LABase.cs`?
* Naming this class as `LoongArchBase`, is it OK ?